Formula/vpn-slice.rb

```diff
@@ -19,16 +19,16 @@ class VpnSlice < Formula
   end
 
   depends_on "rust" => :build # for cryptography, can remove if dnspython has new release
```

Comment:
Can we drop `rust`? I would guess this is because previous version was using `poetry` backend (which has `cryptography` dependency on Linux) rather than `poetry-core`

Reply:
yup I think you're right